### PR TITLE
Remove request virtual card button for unapproved collectives

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/5078 and reported at https://opencollective.slack.com/archives/GFH4N961L/p1641862198006400

no-virtual-card-option

### CODE REVIEW

**Review by znarf — CHANGES_REQUESTED** [review]
The change in itself is fine. 

But I would go extra mile and remove the section from the menu if the Collective is not approved.

**Comment by SudharakaP** [comment]
> The change in itself is fine.
> 
> But I would go extra mile and remove the section from the menu if the Collective is not approved.

Ah yes, that makes sense. I've updated the code to remove the entire section if the collective is not approved (screen-cast updated as well).

File: pages/admin-panel.js [hunk 1]
```diff
@@ -52,6 +52,9 @@ export const adminPanelQuery = gqlV2/* GraphQL */ `
           settings
         }
       }
+      ... on Collective {
+        isApproved
+      }
```
Comment (znarf) [inline pages/admin-panel.js:57]: Why not on `AccountWithHost`? You're missing FUNDs with this approach.

Comment (SudharakaP) [reply pages/admin-panel.js]: Ah, thanks for pointing it out. I've corrected this. 👍🏽

**Comment by znarf** [comment]
@SudharakaP Conflict to resolve.

**Comment by SudharakaP** [comment]
> @SudharakaP Conflict to resolve.

Resolved. 👍🏽

**Comment by znarf** [comment]
What if people go to the /virtual-cards page directly?

**Comment by SudharakaP** [comment]
> What if people go to the /virtual-cards page directly?

Thanks for pointing this out. I saw this issue while working on this but forgot to open a new issue. This happens also for other sections. For example one can go to Tickets or Payment methods etc. I've opened a new issue for this; https://github.com/opencollective/opencollective/issues/5147

no-menu-item

**Review by znarf — APPROVED** [review]

**Pull request merged**